install.yml

```diff
@@ -1,9 +1,7 @@
-$schema: http://json-schema.org/draft-07/schema
-$id: https://adobe.io/schemas/app-builder-templates/1
- 
 categories:
+  - helper-template
```

Comment:
This is for DEVX-2457, "Exclude helper templates from a list of available for installation templates".